libtelemetry/src/main/java/com/mapbox/android/telemetry/Attachment.java

```diff
@@ -0,0 +1,75 @@
+package com.mapbox.android.telemetry;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Attachment extends Event implements Parcelable {
+  private static final String VIS_ATTACHMENT = "vis.attachment";
+
+  @SerializedName("event")
+  private final String event;
+  @SerializedName("attachmentMetadata")
+  private List<AttachmentMetadata> attachmentMetadata;
+  @SerializedName("files")
+  private List<byte[]> files;
+
+  Attachment() {
+    this.event = VIS_ATTACHMENT;
+    attachmentMetadata = new ArrayList<>();
```

Comment:
For consistency sake, would it make sense to use "this.* =" pattern across all constructor initializations?